x-pack/plugins/siem/public/components/open_timeline/search_row/index.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFilterGroup,
+  EuiFilterButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  // @ts-ignore
+  EuiSearchBar,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { OpenTimelineProps } from '../types';
+
+import * as i18n from '../translations';
+
+const ShowingContainer = styled.div`
+  user-select: none;
+`;
+
+const SearchRowFlexGroup = styled(EuiFlexGroup)`
+  user-select: none;
+`;
+
+const FilterGroupFlexItem = styled(EuiFlexItem)`
+  margin-left: 24px;
+`;
+
+const SelectableQueryText = styled.span`
+  user-select: text;
+`;
+
+type Props = Pick<
+  OpenTimelineProps,
+  'onlyFavorites' | 'onQueryChange' | 'onToggleOnlyFavorites' | 'query' | 'totalSearchResultsCount'
+>;
+
+/**
+ * Renders a badge containing the value of `network.direction`
+ */
+export const SearchRow = pure<Props>(
+  ({ onlyFavorites, onQueryChange, onToggleOnlyFavorites, query, totalSearchResultsCount }) => (
+    <>
+      <SearchRowFlexGroup
+        alignItems="flexStart"
+        direction="row"
+        gutterSize="none"
+        justifyContent="spaceBetween"
+      >
+        <EuiFlexItem grow={true}>
+          <EuiSearchBar
+            data-test-subj="search-bar"
+            box={{
+              placeholder: i18n.SEARCH_PLACEHOLDER,
+              incremental: false,
+            }}
+            onChange={onQueryChange}
+          />
+        </EuiFlexItem>
+
+        <FilterGroupFlexItem grow={false}>
+          <EuiFilterGroup>
+            <EuiFilterButton
+              data-test-subj="only-favorites-toggle"
+              hasActiveFilters={onlyFavorites}
+              onClick={onToggleOnlyFavorites}
+            >
+              {i18n.ONLY_FAVORITES}
+            </EuiFilterButton>
+          </EuiFilterGroup>
+        </FilterGroupFlexItem>
+      </SearchRowFlexGroup>
+
+      <EuiSpacer size="s" />
+
+      <ShowingContainer data-test-subj="showing">
+        <EuiText color="subdued" size="xs">
+          <FormattedMessage
+            data-test-subj="query-message"
+            id="xpack.siem.open.timeline.showingNTimelinesLabel"
+            defaultMessage="Showing {totalSearchResultsCount} Timelines {with} "
```

Comment:
You're not using plurals here. So if this has one search result it will be:

```ts
Showing 1 Timelines
```

You can see examples of plural here:
https://github.com/elastic/kibana/tree/master/packages/kbn-i18n#react

Currently from other PR's with other devs doing Plurals the thinking has been that if it is zero you would have:

0 Timlines

If 1, it would be

1 Timeline

If > 1, it would be

n Timelines